### DISCUSSION TITLE
请问能否关闭更新提示

### DISCUSSION THREAD

**Discussion by Songzhe1024** [xfangfang/wiliwili#373] [Q&A]
设备是win11,我使用的是v1.3.0，尝试多次使用最新的版本，但是总出来屏幕花屏之类的问题，不知道问题出在哪，所以想停留在旧版本。但是每次启动wiliwili都会提示更新有些影响使用体验，想请问作者怎么关闭这个更新提示

**Comment by xfangfang** [comment]
@Songzhe1024 可以反馈一下你正在使用的CPU和显卡型号，如果可以的话截图或摄屏看一看“花屏的具体效果”
问题可能和这个issue相关：https://github.com/xfangfang/wiliwili/issues/338

从 1.4.0 开始 wiliwili 默认使用 d3d11 作为图形API（性能提升比较大），不过可以随时在：https://nightly.link/xfangfang/wiliwili/workflows/build.yaml/dev 下载到 gl 后缀的测试版windows客户端，比如：wiliwili-Windows-0087cde-gl-x86_64 gl后缀的客户端使用 OpenGL 3.2+ 作为图形API。

同时也可以在配置文件中设置自定义的更新地址来临时屏蔽更新（设置一个空白的值就可以了）："custom_update_api": "",

关于配置文件在哪，可以查看项目 wiki。

---

当然，我还是建议反馈更多的关于系统的信息。如果有能力，在本地尝试编译看看能否解决这个问题，这个问题可能就是很简单的某个配置造成的，比如这里的初始化部分：https://github.com/xfangfang/borealis/blob/wiliwili/library/lib/platforms/driver/d3d11.cpp

**Reply by Songzhe1024** [reply]
谢谢！我的设备信息：
处理器：英特尔酷睿Ultra5125H
显卡： Intel(R) Arc(TM) Graphics (128MB / 联想)
首先通过您提供的gl版客户端可以正常使用新版。
接着我复现了之前的bug，版本为x86_64的1.4.0,使用截图工具时候花屏会消失只能用手机拍照
![IMG_20240706_192520](https://github.com/xfangfang/wiliwili/assets/141692354/759cfeb9-3ee0-4438-a129-4d8f83d02dee)
![IMG_20240706_192524](https://github.com/xfangfang/wiliwili/assets/141692354/3849fc65-e136-4fd7-a2e7-00e1fbd9e2e0)
![IMG_20240706_192531](https://github.com/xfangfang/wiliwili/assets/141692354/2bbfd156-cf1a-4177-92aa-057c38d55e13)

**Reply by xfangfang** [reply]
感谢反馈，您看我下面的描述哪个对哪个错：
1. 花屏不是一直出现的，只要不进入视频播放页，一直在外面操作就触发不了花屏
2. 进入播放页之后，花屏百分百会出现，但是截屏的图片是正常的，不过肉眼看到的窗口画面是有问题的
3. 进入播放页后，视频区域只要在播放中就没有花屏现象，其他区域可能有花屏
4. 花屏发生时，调整窗口大小或切换全屏（快捷键F）对花屏没有改善
5. 上下滑动屏幕时图片或文字区域一直是正常的，其他位置会有问题

因为前面描述的花屏确实不足以让人了解具体的问题，所以最好可以使用手机录制一下屏幕，从软件刚刚开启时录制，这样如果有其他对D3D比较了解的开发者可能就很容易找到问题的原因了。

**Reply by Songzhe1024** [reply]
1. 是的，只有进入视频播放页才会触发花屏
2. 是的
3. 视频播放中没有花屏，只有视频页面和右侧的评论板块的交界处会有花屏
4. 花屏发生时，调整窗口大小或全屏，花屏会消失
5. 没有这个问题
具体可见视频：

https://github.com/xfangfang/wiliwili/assets/141692354/de45eb22-dbf8-486f-82db-41362a778328

**Reply by xfangfang** [reply]
感谢反馈，看起来像是播放器组件的问题，这个我也不太懂具体的原因，如果你对这个问题感兴趣的话，我可以指出来用到的播放器代码的位置，您可以在本地尝试自己改改看。不然的话就只能先用这个OpenGL的版本了。

**Reply by Songzhe1024** [reply]
谢谢作者，我先使用OpenGL版本了。解决这个问题对来我来太难了目前:)

**Reply by dragonflylee** [reply]
@Songzhe1024 可以尝试更新一下最新的核显驱动

https://www.intel.cn/content/www/cn/zh/download/741626/intel-arc-pro-graphics-windows.html

**Reply by Songzhe1024** [reply]
> @Songzhe1024 可以尝试更新一下最新的核显驱动
> 
> https://www.intel.cn/content/www/cn/zh/download/741626/intel-arc-pro-graphics-windows.html



> > 谢谢作者，我先使用OpenGL版本了。解决这个问题对来我来太难了目前:)
> 
> @Songzhe1024 有空的时候帮忙测试一下这个版本还会不会花屏 https://nightly.link/dragonflylee/wiliwili/actions/runs/9833285915/wiliwili-Windows-859503a-d3d-x86_64.zip

测试依旧花屏

**Reply by dragonflylee** [reply]
@Songzhe1024  有空帮忙再测测这个修改版会不会花屏，如果不行的化短时间内就没啥好办法了

https://nightly.link/dragonflylee/wiliwili/actions/runs/9850174147/wiliwili-Windows-ceaa392-d3d-x86_64.zip

**Reply by Songzhe1024** [reply]
@dragonflylee 还是没有改善
